cli.js

```diff
@@ -23,44 +30,36 @@ Options:
 
 const options = docopt(doc, { version: '0.0.1' });
 
-try {
-    if(options['setup']){
-        console.log("Running setup")
-        runSetup(options['<parsecdLocation>'])
-    }
+if(!global_state.config["parsecdFound"]){
+    console.log("Parsecd not found, run \"parsec-switcher setup <parsecdLocation>\" ")
+}
+else {
+    if (options['-a'] || options['--add']){
+
+        console.log(`Adding ${options['<nickname>']}`);
+        addAccount(options['<nickname>']);
 
-        if(setupRequired()){
-            if(!autoSetup()){
-                throw new Error("Setup required, run parsec-switcher setup \"Your Parsecd.exe location\", this will nuke your parsec-switcher data and accounts")
-            }
-        }
+    }
+    if (options['-d'] || options['--delete']){
+        console.log(` Deleting account ${options['<nickname>']}`);
+        deleteAccount(options['<nickname>']);
+    }
+    if (options['-s'] || options['--switch']){
+        console.log(`Switching account to ${options['<nickname>']}`);
+        switchAccount(options['<nickname>'])
 
+    }
+    if (options['-l'] || options['--list']){
+        console.log('Printing the list ')
+        list = returnAccountList()
```

Comment:
implicit declaration, maybe use `const` before it

Reply:
done